apps/storefront/pages/monstre/feilmeldinger.mdx

```diff
@@ -4,59 +4,112 @@ import { Figma } from 'mdx-embed';
+
+## Andre eksempler på denne komponenten
+Feilmeldinger brukes i mange komponenter og kontekster. Hvordan vi viser og utformer feilmeldingen kan variere litt fra kontekst til kontekst, se eksemplene under. 
+
+### Feilmeldinger med accordion (trekkspilliste)
+[Eksempel]
+
+### Feilmeldinger med date field (datofelt)
+[Eksempel]
+
+### Feilmeldinger med drop-down menu (nedtrekkesmeny)
+[Eksempel]
 
-- “Postnummer må ha 4 siffer”
-- "Fødselsnummer må bestå av fødselsdato (6 siffer) og personnummer (5 siffer)"
-- “Du må velge minst ett av leveringsalternativene”
-- “For å sende inn skjemaet må du bekrefte at navnet er korrekt ved å huke av i avkrysningsboksen”
+### Feilmeldinger med date picker (datovelger)
+[Eksempel]
 
-### Eksempel på feilmelding relatert til avkrysningsbokser
+### Feilmeldinger med input field (inndatafelt)
+[Eksempel]
 
-I dette eksempelet har brukeren forsøkt å trykke “Neste” uten å svart på obligatoriske spørsmål. Den røde boksen i bunnen viser kun når “Neste” er forsøkt klikket på.
+### Feilmeldinger med radio buttons (alternativknapper)
+[Eksempel]
+
+### Feilmeldinger med text field (tekstfelt)
+[Eksempel]
+
+### Feilmeldinger med checkboxes (avmerkingsbokser)
+
+Her har brukeren trykket på “Neste” uten å ha svart på spørsmål som må fylles ut. Den røde boksen i bunnen vises bare hvis brukeren har trykket på “Neste”.
 
 <Figma
   title='Boop'
   url='file/vpM9dqqQPHqU6ogfKp5tlr/Felles-komponenter?type=design&node-id=17866%3A57910&mode=design&t=3hD9xEYGKMGecg7Q-1'
 />
 
-### Plassering av feilmelding
+### Feilmeldinger når brukerne skal legge ved en fil (attach)
+[Eksempel]
+
+
+## Retningslinjer for når du skal bruke feilmeldinger
+
+Bruk feilmeldinger til å hjelpe brukeren gjennom de stegene de skal gjennom i en tjeneste.
+
+Ikke bruk feilmeldinger til å fortelle brukerne at det er noe de ikke har rett på, eller ikke tillatelse til å gjøre. Hvis du trenger å gi denne typen informasjon, er det best å vise den på en egen side, og gi dem opplysninger om hva de kan gjøre videre.
+
+Ikke bruk feilmeldinger til å varsle brukeren om feil som ikke kan forbindes med noe med brukeren gjør, for eksempel systemfeil. Da bruker vi heller alert-komponenten.
+
+### Tekst i feilmeldinger
+
+Den viktigste jobben til teksten i en feilmelding er å hjelpe brukerne videre:
+- Tenk på målgruppen og skriv vennlig, klart og tydelig. Unngå passivt eller teknisk språk. Meldingen skal være lett å forstå. 
+- Unngå humor i feilmeldinger. Når folk møter på et problem, vil de heller ha effektive forslag til hvordan de løser det, enn humoristiske «Oops!»
+- Vær konkret. Mange feilmeldinger kan være for generelle og fortelle at «noe gikk galt». 
+- Noen ganger må vi beskrive hva feilen er før vi beskriver hva brukerne skal gjøre, men det viktigste er å vise løsningen på problemet.
+- Tilpass feilmeldingene til ulike situasjoner. Jobben til teksten er å hjelpe brukeren videre.
+- Hold feilmeldingen kort, da sparer du brukerne for tid.
+
```

Comment:
Burde vi legge til et punkt om at teksten i feilmeldingen må inneholde en referanse til ledeteksten i spørsmålet. Alle eksemplene du har laget nedenfor er gode eksempler på dette, men har sett at det ofte ikke blir tatt med i feilmeldinger. For eksempel: "Feltet er påkrevd", i stedet for "epost må fylles ut".